src/Storage/Storage/Common/Cmdlet/NewAzureStorageContext.cs

```diff
@@ -178,6 +178,16 @@ public SwitchParameter Anonymous
 
         private bool isAnonymous;
 
+        [Parameter(HelpMessage = "Use OAuth storage account", Mandatory = false, ParameterSetName = SasTokenParameterSet)]
+        public SwitchParameter SASTokenWithConnectedAccount
```

Comment:
Why add a new parameter, but not just add the exist "-UseConnectedAccount" to the SAS parameter set?
Normally we should keep as less parameters as possible, and the 2 parameters actually do same thing: add token to the context.